lib/charms/opensearch/v0/opensearch_base_charm.py

```diff
@@ -405,18 +420,27 @@ def _can_service_start(self):
 
         return True
 
-    def _initialize_admin_user(self):
+    def _initialise_internal_users(self):
```

Comment:
Could we keep this `admin` related? and have the user cleanup on another method `_remove_default_users` that gets called before this one? because this method will be called by the password rotate action - and it won't be very clear